### PR TITLE
RoyalApps.RoyalTS version 6.1.50425

### PR DESCRIPTION
**Pull request by StefanKoell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58803)

### CODE REVIEW

File: manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.50425
```
Comment (OfficialEsco) [inline manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml:5]:
```suggestion
PackageVersion: 6.1.50425.0
```

File: manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.50425
```
Comment (OfficialEsco) [inline manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.1.50425.0
```

File: manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.50425
```
Comment (OfficialEsco) [inline manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.yaml:5]:
```suggestion
PackageVersion: 6.1.50425.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
Name          : Royal TS 6.01.50425.0
Publisher     : Royal Apps GmbH
Version       : 6.1.50425.0
ProductCode   : {2408F291-807B-43A8-9731-61C35F096E1B}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{2408F291-807B-43A8-9731-61C35F096E1B}
Uninstall     : winget uninstall --id "{2408F291-807B-43A8-9731-61C35F096E1B}"
```

File: manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml (hunk 1 above)
Comment (StefanKoell) [reply manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml:5]: I think this is a problem with wingetcreate command line. I specified the version 6.1.50425 (without the .0) but the installer is built (for other reasons) with the .0 at the end. I can commit this change for sure but I'm afraid the next time it runs in my pipeline, we have the same issue. Should I open an issue with wingetcreate for that?

Comment (OfficialEsco) [reply manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml]: Your installer includes .0 in the DisplayVersion registry string, so if you specified without the .0 to wingetcreate then.. you are the issue?

Afaik wingetcreate uses your specified values unless its a new package.

**Comment by StefanKoell** [comment]
Not sure what I should do now. I used wingetcreate to submit the manifest. If it's not compliant, I'm not sure what else I can do.

**Comment by OfficialEsco** [comment]
Hopefully you saw https://github.com/microsoft/winget-pkgs/pull/58803#discussion_r858509271
Current issue:
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\r\RoyalApps\RoyalTS\6.1.50425', 
expected 'manifests\r\RoyalApps\RoyalTS\6.1.50425.0'
```
Press `.` to open VSCode Online and rename the version folder name from `6.1.50425` to `6.1.50425.0`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/58803#issuecomment-1109592060

File: manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml (hunk 1 above)
Comment (StefanKoell) [reply manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml]: Correct, my bad. Will update my script.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**